metadata/SignatureComparison__1__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "SignatureComparison",
+    "retrainable": false,
+    "gpu": 0,
+    "processorType": "CPU",
+    "description": "The UiPath signature comparison model is used to determine the similarity between a pair of signature and suggest if the signatures are from the same author.",
+    "inputDescription": "A pair of signature images in a list. Files formats accepted are png, and jpeg.",
+    "outputDescription": "Similarity score (between 0 and 1) and a preliminary determination if the signatures are from the same author or different authors based on a defined threshold in a Json format.",
+    "changeLog": "",
+    "cpu": 1,
+    "inputType": "FILE_LIST",
+    "displayName": "SignatureComparison",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38_OPENCV",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Image Analysis",
+    "projectDescription": "Curated Models from UiPath to analyze images",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
```

Comment:
@vivekjhauipath This is the right format for min aicenter version when referring to 21.10?

Reply:
yes, we removed v from 22.4 onwards